Drop-In/src/main/java/com/braintreepayments/api/DropInPaymentMethodType.java

```diff
@@ -72,7 +72,7 @@ public static DropInPaymentMethodType forType(PaymentMethodNonce paymentMethodNo
      * @return a {@link CardType[]} containing the mapped {@link CardType}s from the supplied
      * supportedCardTypes.
      */
-    public static CardType[] getCardsTypes(Set<String> supportedCardTypes) {
+    public static CardType[] getCardsTypes(List<String> supportedCardTypes) {
```

Comment:
Curious why the switch here. We want the order that the List provides?

Reply:
We've moved away from `Set` in our SDK interfaces a bit. There aren't any huge benefits with `Set` that we're taking advantage of. We still enforce uniqueness internally then convert the set into a list.